stencil-library/src/components/table/table.scss

```diff
@@ -8,4 +8,13 @@
 
 :host {
   @include host-base-styles;
+  table-layout: fixed;
 }
+
+.table-wrapper {
+  overflow-x: scroll;
+}
+
+.table-light {
```

Comment:
I think we could try to use a bootstrap class like `.text-nowrap` here. Or create a new class to add this style, the reason being that bootstrap uses `table-light` to style the table, and if we decide to change to `table-dark` we will have to update the CSS too.

Reply:
Great call - thanks for the suggestion, and explanation! 

Just updated it to simply apply the `.text-nowrap` class as you suggested, no need to create a new class here I think, or modify `table-light`.